server/server.go

```diff
@@ -272,8 +275,15 @@ func configureStorage(logger *logrus.Logger, conf *Config) (storage Storage, err
 	if conf.RedisURL != "" {
 		return NewRedisStorage(conf.RedisURL)
 	}
+	if conf.SqliteURL != "" {
+		return NewSqliteStorage(conf.SqliteURL)
+	}
+	if conf.Snapshot {
```

Comment:
should we check and err if snapshot is configured but also the ":memory:" since they are exclusive?

Reply:
I was thinking snapshot overrides memory, as the default is ":memory:" otherwise with single 'snapshot' flag we would error, but I can change the default also.

Reply:
I feel if the user provides both they should be maybe warned, not sure. I will let you decide, just pointed out.

Reply:
agreed I misunderstood at first, if the filename is provided we already should disable the snapshot.

Reply:
fixed in ca045c3